src/ventus/wm/window.js

```diff
@@ -322,13 +324,23 @@ function(Emitter, View, WindowTemplate) {
 					this.el.hide();
 
 					// Remove element
-					this.$content.html('');
+					if(!this.softRemove) {
```

Comment:
@macqm This is typical UI problem, it is usually handled with ```preventDefault``` calls in the events. So the close action should emit two events, the classic ```beforeClose``` and ```close`` events. The default behavior for close should be to destroy the window, unless the ```beforeClose``` event is captured and the user's code actually calls a ```event.preventDefault()``` method.

The problem here is that the events library I was using doesn't support ```preventDefault``` and the beforeClose event should be created.